invenio_app_ils/config.py

```diff
@@ -741,6 +748,31 @@ def _(x):
         update_permission_factory_imp=backoffice_permission,
         delete_permission_factory_imp=backoffice_permission,
     ),
+    apordid=dict(
+        pid_type=ACQUISTION_PURCHASE_ORDER_PID_TYPE,
+        pid_minter=ACQUISTION_PURCHASE_ORDER_PID_MINTER,
+        pid_fetcher=ACQUISTION_PURCHASE_ORDER_PID_FETCHER,
+        search_class=AcquisitionPurchaseOrderSearch,
+        record_class=AcquisitionPurchaseOrder,
+        record_serializers={
+            'application/json': ('invenio_records_rest.serializers'
+                                 ':json_v1_response'),
+        },
+        search_serializers={
+            'application/json': ('invenio_records_rest.serializers'
+                                 ':json_v1_search'),
+        },
+        list_route='/acquisition/acquisition-purchase-orders/',
```

Comment:
Shall we go for something shorter? `/acquisition/orders` and `/acquisition/orders/1`